en/lessons/advanced/escripts.md

```diff
@@ -26,6 +39,7 @@ end
 Next we need to update our Mixfile to include the `:escript` option for our project along with specifying our `:main_module`:
 
 ```elixir
+# ./mix.exs
```

Comment:
Please remove this, we do not use this format in any other lessons and we like to remain consistent.